### PR TITLE
Add Docker Compose instructions.

### PR DESCRIPTION
**Pull request by blackpioter**
Fixes issue #135 
Changes: README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This is the official website of the Programming Club at UIET, Panjab University.
 ### Setup your development environment
 1. Fork this repository to your account.
 
-2. Create a virtual environment on your machine. 
+2. Create a virtual environment on your machine.
```
Comment (divyam3897) [inline README.md]: Remove this change.

File: README.md [hunk 2]
```diff
@@ -25,30 +25,57 @@ This is the official website of the Programming Club at UIET, Panjab University.
     ```
     git clone https://github.com/pclubuiet/website.git
     ```
-    
+
```
Comment (divyam3897) [inline README.md]: Remove this change

File: README.md [hunk 3]
```diff
@@ -25,30 +25,57 @@ This is the official website of the Programming Club at UIET, Panjab University.
...
 5. Install the dependencies for the project.
     ```
     cd website
     pip3 install -r requirements.txt
     ```
-    
+
```
Comment (divyam3897) [inline README.md]: Remove the change

File: README.md [hunk 4]
```diff
@@ -25,30 +25,57 @@ This is the official website of the Programming Club at UIET, Panjab University.
...
 6. Run the migrations and collect static files.
     ```
     python3 manage.py migrate
     python3 manage.py collectstatic
     ```
-    
+
```
Comment (divyam3897) [inline README.md]: Remove the change

File: README.md [hunk 5]
```diff
@@ -25,30 +25,57 @@ This is the official website of the Programming Club at UIET, Panjab University.
...
 7. Run the live development server on your machine and test it.
     ```
     python3 manage.py runserver
     ```
     Once the server is started, open http://127.0.0.1:8000/home in a web browser.
     Everything went well if the webpage loads correctly and you don't see any errors.
-    
+
```
Comment (divyam3897) [inline README.md]: Remove the change

File: README.md [hunk 6]
```diff
@@ -78,7 +105,7 @@ After selecting an issue
     Then stage them and commit them.
     Check out Chris Beams's guide to writing good commit messages [here](https://chris.beams.io/posts/git-commit/).
 
-    *A small description of your changes is must in the commit messages.* 
+    *A small description of your changes is must in the commit messages.*
```
Comment (divyam3897) [inline README.md]: Remove this change

**Comment by blackpioter** [comment]
@divyam3897 can you explain why white characters should be restored at EOL ?

**Comment by divyam3897** [comment]
They should not be but that should not be done in this PR, you can create another issue for the same first and then it can be another beginner friendly issue for someone finding their first issue :)

**Comment by blackpioter** [comment]
@divyam3897 I've updated PR according to your requirements.

**Comment by blackpioter** [comment]
@divyam3897 have you checked the changes you requested?

**Comment by divyam3897** [comment]
also as a general practice you should never work in master, please follow the README.

**Comment by blackpioter** [comment]
@divyam3897 I'll close this PR and open a new one.

**Pull request closed without merging**